platform/src/test/java/org/exoplatform/platform/qa/ui/platform/chat/ChatManageMessageTestIT.java

```diff
@@ -223,4 +238,77 @@ public void test04_saveMeetingNote() {
     wikiHomePage.deleteWiki(nameWikipage);
 
   }
+
+  @Test
+  public void test05_checkTimeWhenSendAmessageInChatRoom(){
+    String room = "room" + getRandomNumber();
+    String message="message"+getRandomNumber();
+    String username = "username" + getRandomNumber();
+    String password = "123456";
+    String FirstName = "FirstName" + getRandomString();
+    String LastName = "LastName" + getRandomString();
+    String email = username + "@test.com";
+    String I;
+
+    navigationToolbar.goToAddUser();
+    userAddManagement.addUser(username, password, email, FirstName, LastName);
+    homePagePlatform.goToChat();
+    switchTo().window(1);
+    roomManagement.addRoom(room, username);
+    chatManagement.sendMessageInRoomOrSpace(room,message);
+    I=$(byClassName("message-time")).waitUntil(Condition.appear, Configuration.timeout).getText();
```

Comment:
1- Variable must have meaningful name.
2- you must specify which message you will get its time using $(byText(message)).parent()....

Reply:
We find only a message time after add a room and send a message